### DISCUSSION TITLE
Welcome to Nellik-s-HighDef-GUIs Discussions!

### DISCUSSION THREAD

**Discussion by MeatSh1eld** [MeatSh1eld/Nellik-s-HighDef-GUIs#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * And put mod request in [Issuses](https://github.com/MeatSh1eld/Nellik-s-HighDef-GUIs/issues)

**Comment by jusitnboggs** [comment]
Any plans for an update to 1.18, 1.19, and 1.20?
